app/messages/service_credential_bindings_list_message.rb

```diff
@@ -24,5 +24,9 @@ class ServiceCredentialBindingsListMessage < ListMessage
     def self.from_params(params)
       super(params, ARRAY_KEYS.map(&:to_s))
     end
+
+    def valid_order_by_values
+      super << :name
```

Comment:
```suggestion
      super + [:name]
```

Not sure if this will actually matter since I think we freeze most constants, but `<<` modifies and return new value while `+` will just return a new value.